### PR TITLE
Versioned docs

### PR DESCRIPTION
**Pull request by 2can**
This PR introduces Versioned docs.
- The current `/docs/3.x` has been versioned and is available under `versioned_docs` folder
- the docs folder is available as the `next` version and available under `/docs/next` path

The changes made to the main `docs` folder will be available in the `next` version.
The changes made to the `versioned_docs/version-3.x` folder will be available in the `3.x` version (current).

Changes for the future versions of Saleor should be made in `docs` folder.
Changes for the current docs should be applied both in `docs` & `versioned_docs/version-3.x` folders.

### CODE REVIEW

**Comment by korycins** [comment]
@2can  Question. Isn't something that should be worth mentioning in README?

**Comment by korycins** [comment]
Question2: How can I switch between `3.X` and `next` version? I don't see any version switcher on the preview build 🤔

**Review by korycins — APPROVED** [review]

File: docusaurus.config.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 require("dotenv").config();
 
+const isUpdate = process.env.UPDATE_SALEOR;
```
Comment (mmiszy) [inline docusaurus.config.js:3]: What does it mean `isUpdate` and `UPDATE_SALEOR`?
Is it documented anywhere?

Comment (2can) [reply docusaurus.config.js:3]: It's used in the `Releasing Canary to 3.x` section in README and it allows properly re-create `3.x` based on `Canary`

**Review by patrys — APPROVED** [review]

**Pull request merged**